lib/brcobranca/boleto/bradesco.rb

```diff
@@ -11,6 +11,9 @@ class Bradesco < Base # Banco BRADESCO
       # @param (see Brcobranca::Boleto::Base#initialize)
       def initialize(campos = {})
         campos = { carteira: '06' }.merge!(campos)
+
+        campos.merge!(local_pagamento: 'Pagável preferencialmente na Rede Bradesco ou Bradesco Expresso')
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.